pkg/device_plugin/generic_device_plugin.go

```diff
@@ -59,7 +59,8 @@ type GenericDevicePlugin struct {
 	devs       []*pluginapi.Device
 	server     *grpc.Server
 	socketPath string
-	stop       chan struct{}
+	stop       chan struct{} // this channel signals to stop the DP
+	term       chan bool     // this channel detects kubelet restarts
```

Comment:
Do we also want `term` in the vgpu device plugin?

Reply:
Yes, forgot to push this change. Pushed now.